pkg/network/common/egressip.go

```diff
@@ -151,11 +151,11 @@ func NewEgressIPTracker(watcher EgressIPWatcher, cloudEgressIP bool, localIP str
 
 func (eit *EgressIPTracker) Start(kubeClient kubernetes.Interface, hostSubnetInformer osdninformers.HostSubnetInformer, netNamespaceInformer osdninformers.NetNamespaceInformer, nodeInformer kcoreinformers.NodeInformer) {
 
+	eit.kubeClient = kubeClient
```

Comment:
FYI: there is no need to make this assignment conditional. If `nil` is passed in, it shall remain as `nil`.